From fed7b5574acfe42f6c402dabeee0e3b55550274f Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 20 Dec 2024 15:58:04 +0100
Subject: [PATCH] fixup! arm64: dts: qcom: pm8150b: Add a charger
 Signed-off-by: Teguh Sobirin <teguh@sobir.in>

Use constants for io-channels properties, and make names property be one
value per line, as is common in dts.
---
 arch/arm64/boot/dts/qcom/pm8150b.dtsi | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/pm8150b.dtsi b/arch/arm64/boot/dts/qcom/pm8150b.dtsi
index efad258834df..994b24c50f0a 100644
--- a/arch/arm64/boot/dts/qcom/pm8150b.dtsi
+++ b/arch/arm64/boot/dts/qcom/pm8150b.dtsi
@@ -121,9 +121,10 @@ pm8150b_charger: charger@1000 {
 					  "wdog-bark",
 					  "usbin-icl-change";
 
-			io-channels = <&pm8150b_adc 7>,
-				      <&pm8150b_adc 8>;
-			io-channel-names = "usb_in_i_uv", "usb_in_v_div_16";
+			io-channels = <&pm8150b_adc ADC5_USB_IN_I>,
+				      <&pm8150b_adc ADC5_USB_IN_V_16>;
+			io-channel-names = "usb_in_i_uv",
+					   "usb_in_v_div_16";
 
 			status = "disabled";
 		};
-- 
2.34.1

